server/src/main/java/org/apache/druid/discovery/DruidLeaderClient.java

```diff
@@ -120,6 +120,17 @@ public void stop()
     log.info("Stopped.");
   }
 
+  /**
+   * Make a Request object aimed at the leader. Throws IOException if the leader cannot be located.
+   *
+   * @param cached Uses cached leader if true, else uses the current leader
+   */
+  public Request makeRequest(HttpMethod httpMethod, String urlPath, boolean cached) throws IOException
```

Comment:
Please fix the below `makeRequest` to call this.

Reply:
ok, done